f600233d1b154d507b782f84604bcb12/update/sys_hub_action_type_definition_738c03a4979211106fa8b4b3f153af63.xml

```diff
@@ -150,14 +150,26 @@
         <sys_created_by>julius.fitzhugh-ctr</sys_created_by>
         <sys_created_on>2022-09-29 17:09:21</sys_created_on>
         <sys_id>0076d3ac979211106fa8b4b3f153af25</sys_id>
-        <sys_mod_count>31</sys_mod_count>
-        <sys_updated_by>1370228783.CTR</sys_updated_by>
-        <sys_updated_on>2022-10-14 15:43:04</sys_updated_on>
+        <sys_mod_count>32</sys_mod_count>
+        <sys_updated_by>admin</sys_updated_by>
+        <sys_updated_on>2023-08-01 18:30:45</sys_updated_on>
         <value>(function execute(inputs, outputs) {
   var reqUtil = new Request()
   var errUtil = new ErrorHandler()
   var response;
  
+  var token = inputs.token
+  if (inputs.credential_id) {
+    try {
+      var provider = new sn_cc.StandardCredentialsProvider();
+      var credential = provider.getCredentialByID("x_g_dis_atat.credentials." + inputs.credential_id);
+      if (credential) {
+	      token = credential.getAttribute("password")
+      }
+    } catch (error) {
```

Comment:
We should at least log a warning if the given `credential_id` could not be found, particularly since we are doing some additional logic that folks might not be expecting.

Reply:
It's catches the exception and logs it. What more would you like to see?